src/nodes/ImportDirective.js

```diff
@@ -3,7 +3,8 @@ const {
     builders: { group, line, softline }
   }
 } = require('prettier');
-const semver = require('semver');
+const coerce = require('semver/functions/coerce');
+const satisfies = require('semver/functions/satisfies');
```

Comment:
Can we destructure them in one line?

Reply:
we are picking from different files. if we destructure, we load the whole server